react/src/lib/components/SubsurfaceViewer/layers/map/mapLayer.ts

```diff
@@ -172,7 +184,7 @@ export interface MapLayerProps<D> extends ExtendedLayerProps<D> {
      * each direction then the property values will be pr cell and the cell will be constant
      * colored.
      */
-    propertiesUrl: string;
+    propertiesUrl: string | number[];
```

Comment:
If properties now can be not only URL, is the name "propertiesUrl" still appropriate?

Also, is it possible to use the Float32Array directly in order to enforce 32 bit float from the client directly, which could avoid confusion caused by truncation, say if double precision was used?